controllers/companies/edit/get.js

```diff
@@ -0,0 +1,21 @@
+const Company = require('../../../models/company/Company');
+
+module.exports = (req,res) =>{
+  const company_id = req.query["id"];
```

Comment:
Burda böyle yapmasan olur, app.js de farkındaysan req.query yoksa boş objeye eşitliyorum. Direk req.query.id diyerek çağırabilmen için o fonksiyonu. Hani çok da fark etmez ama o satır olmasa da olur

Reply:
O satırı sadece okunabilirliği arttırmak için yazdım. IDyi aldığımı belirtmek için yani.